vmm/sandbox/Cargo.toml

```diff
@@ -7,12 +7,6 @@ edition = "2021"
 [profile.release]
 panic = 'abort'
 
-[features]
-default = []
-qemu = ["qapi/qmp", "qapi/async-tokio-all", "unshare", "os_pipe"]
```

Comment:
Please try to add the mapping of binary name and its main.rs path. https://doc.rust-lang.org/cargo/reference/cargo-targets.html#configuring-a-target

Reply:
I added configuration for qemu, cloud_hypervisor, stratovirt.